gzap.go

```diff
@@ -48,6 +48,12 @@ func initLogger(cfg Config, disableGraylog bool) error {
 		return setGraylogLogger(cfg, zapcore)
 	}
 
+	// if ENABLE_DATADOG_JSON_FORMATTER then replace
+	// zapcore console log with json production log
+	if cfg.enableGrayLogJSONFormatter() {
+		return setJSONFormatterLogger()
```

Comment:
@andela-sjames Quick Q. So if we enable the JSON formatter, does that mean we can't also use the graylog setup?

Reply:
Hmmm, from my implementation that's what would happen. I suppose we don't want that yet, rather we should have both enabled?